### PR TITLE
Optimize

### PR DESCRIPTION
**Pull request by T-Damer**
- Optimized sc-aniamtion block to change only when visible
- Optimized svgs to use less code
- Removed unnecessary `dropShadow` and used `boxShadow`
- Fixed bg-blur visibility for FF on third block
- Optimized transformations by applying `will-transform`
- Reduced `tailwind.config`
- Used `preact` imports instead of `react` imports
- Used `.webp` instead of `.svg` background, `svg` bg (even small parts of it) caused insane lags for ff. 2 webp images weight around 2mb

### CODE REVIEW

File: src/icons/Burger.tsx [hunk 1]
```diff
@@ -1,14 +1,5 @@
-import classnames, {
-  boxShadow,
-  dropShadow,
-  stroke,
-  strokeWidth,
-} from 'classnames/tailwind'
+import classnames, { stroke, strokeWidth } from 'classnames/tailwind'
 
-const svgShadow = classnames(
```
Comment (T-Damer) [inline src/icons/Burger.tsx:8]: It's almost invisible, so I removed it

File: src/icons/GemWithArc.tsx [hunk 2]
```diff
@@ -13,13 +13,13 @@ const textStyle = classnames(
...
-  fontSize('text-3xl', 'md:text-xl')
+  fontSize('text-3xl', 'md:text-xl'),
+  willChange('will-change-transform')
 )
 const svgBox = classnames(
   height('md:h-80', 'h-44'),
   margin('mx-auto', '-mb-4'),
-  scale('scale-200'),
-  willChange('will-change-auto')
+  scale('tiny:scale-175')
```
Comment (T-Damer) [inline src/icons/GemWithArc.tsx:22]: Fixes this clipped text
Icz7d0Syna

File: src/icons/GemWithArc.tsx [hunk 3]
```diff
@@ -78,7 +78,7 @@ export default function ({ text }: ArcTextProps) {
         y={(size - labelSize) / 2}
         width={labelSize}
         height={labelSize}
-        href="/img/comming-soon.svg"
+        href="/img/coming-soon.svg"
```
Comment (T-Damer) [inline src/icons/GemWithArc.tsx:81]: Misspell

File: src/icons/GemWithArc.tsx (hunk 2 above)
Comment (T-Damer) [reply src/icons/GemWithArc.tsx:22]: Fixed positioning for this as well

File: src/hooks/useScrollPercent.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { useCallback, useMemo, useState } from 'react'
+import { useCallback, useMemo, useState } from 'preact/hooks'
 import useThrottle from 'hooks/useThrottle'
 
 export default function () {
```
Comment (T-Damer) [inline src/hooks/useScrollPercent.ts:4]: We don't need two different hooks with almost same logic, we can detect if user is not at the top with this one as well

File: src/helpers/scrollAnimationProvider.ts [hunk 5]
```diff
@@ -6,23 +6,25 @@ import classnames, {
 import useBreakpoints from 'hooks/useBreakpoints'
 import useScrollPercent from 'hooks/useScrollPercent'
 
-export default function (animationName: string) {
+export default function (animationName: string, blockVisible = true) {
```
Comment (T-Damer) [inline src/helpers/scrollAnimationProvider.ts:9]: We remove the `style={}` if the block is not visible, this saves a lot of memory and gpu

File: src/components/SealCredAnimation.tsx [hunk 6]
```diff
@@ -154,9 +154,10 @@ const PairOfNfts = ({ colors, position, small }: PairOfNftsProps) => {
 export default function () {
   const { ref: zkWrapRef, inView: zkWrapVisible } = useInView()
   const { ref: suitWrapRef, inView: suitWrapVisible } = useInView()
+  const { ref: wrapperRef, inView: wrapperVisible } = useInView()
```
Comment (T-Damer) [inline src/components/SealCredAnimation.tsx:157]: Explained it in `useScrollPercent` hook

File: src/components/Root.tsx [hunk 7]
```diff
@@ -14,7 +14,7 @@ const innerBody = classnames(
   alignItems('items-center'),
   margin('mx-auto'),
   position('relative'),
-  overflow('overflow-hidden')
+  overflow('md:overflow-visible', 'overflow-hidden')
```
Comment (T-Damer) [inline src/components/Root.tsx:17]: This broke the `sticky` of left `sc` block, fixed it

File: src/components/GemBlock.tsx [hunk 8]
```diff
@@ -13,7 +13,7 @@ import classnames, {
 } from 'classnames/tailwind'
 import useBreakpoints from 'hooks/useBreakpoints'
 
-const gemCard = space('-space-y-44')
+const gemCard = space('-space-y-48')
```
Comment (T-Damer) [inline src/components/GemBlock.tsx:16]: Fixed clipped `ArcText` and block positioning for it (we used `scale`, so it got broken, explained it more in `GemWithArc.tsx`)

File: tailwind.config.js [hunk 9]
```diff
@@ -2,14 +2,6 @@
 module.exports = {
```
Comment (T-Damer) [inline tailwind.config.js:2]: Removed all unused things from here

File: src/components/OrbSeal.tsx [hunk 10]
```diff
@@ -1,8 +1,9 @@
+import { dropShadow, strokeWidth } from 'classnames/tailwind'
 import classNamesToString from 'helpers/classNamesToString'
-import classnames, { strokeWidth } from 'classnames/tailwind'
 import useBreakpoints from 'hooks/useBreakpoints'
 
-const mainStroke = classnames(strokeWidth('stroke-1.5'))
+const mainStroke = strokeWidth('stroke-1.5')
+const orbShadow = dropShadow('drop-shadow-accent')
```
Comment (T-Damer) [inline src/components/OrbSeal.tsx:6]: Removed filter and used `dropShadow`, looks like they're pretty same talking about optimization, but this has less lines :)

File: src/components/Navbar.tsx [hunk 11]
```diff
@@ -27,7 +26,7 @@ import classnames, {
 } from 'classnames/tailwind'
 import useBreakpoints from 'hooks/useBreakpoints'
 import useClickOutside from 'hooks/useClickOutside'
-import useScrollPosition from 'hooks/useScrollPosition'
+import useScrollPercent from 'hooks/useScrollPercent'
```
Comment (T-Damer) [inline src/components/Navbar.tsx:29]: I guess we don't need two almost same hooks, let's use one

File: src/icons/PrismOrb.tsx [hunk 12]
```diff
@@ -27,147 +27,76 @@ export default function () {
...
-            />
-          </g>
-          <g style={{ mixBlendMode: 'soft-light' }}>
-            <path
-              d="M197.094 160.184L122.393 160.184L122.393 86.7999L197.094 86.7999L197.094 160.184Z"
-              fill="url(#paint6_radial_81_856)"
-            />
-          </g>
-        </g>
+      <circle
```
Comment (T-Damer) [inline src/icons/PrismOrb.tsx:37]: I exported the `svg` without `dropShadow` and used our own `dropShadow` (which is above in `svgClasses`). Result:
image
was: 
image

Comment (T-Damer) [reply src/icons/PrismOrb.tsx:37]: I fixed this black outline

File: src/index.css [hunk 13]
```diff
@@ -56,7 +56,7 @@ body {
 .grid {
   position: absolute;
   width: 300%;
-  left: -75%;
+  left: -100%;
```
Comment (T-Damer) [inline src/index.css:59]: It was a little off-center on big screens: 
image
Now: 
image

**Review by backmeupplz — APPROVED** [review]

**Pull request merged**